spec/root_spec.rb

```diff
@@ -7,6 +7,6 @@
   it 'shows list of first 10 shows' do
     assign :shows, @shows = CSV.parse(File.read('lib/netflix_titles.csv'), headers: true).first(10)
     render
-    @shows.each { |show| expect(rendered).to have_content(show["title"]) }   
+    @shows.each { |show| expect(rendered).to have_content(show['title']) }
```

Comment:
`spec/root_spec.rb` you need a category i.e. `spec/controller/shows_spec.rb`

Reply:
Done